registration/tests/test_approval.py

```diff
@@ -126,9 +128,21 @@ def test_first_appserver_is_active(self):
         Check if the the first Appserver is correctly activated even when
         spawned manually
         """
+        user = get_user_model().objects.create_user(username='test', email='test@example.com')
+
+        UserProfile.objects.create(
+            user=user,
+            full_name="Test user 1",
+            accepted_privacy_policy=datetime.now(),
+            accept_domain_condition=True,
+            subscribe_to_updates=True,
+        )
+
+        user.refresh_from_db()
```

Comment:
This code is being re-used 3 times. There's two options:
1. Can you make a `BetaTestUserFactory` (or similar) on `test/utils.py` and use that instead?
2. Use the https://github.com/open-craft/opencraft/blob/master/registration/tests/utils.py#L181 test helper (with a few minor changes to also add the user profile).

I prefer option 1, but I want to know which one do you think is cleaner?

Reply:
I went ahead with option 1 because of three reasons:

1. UserMixin is currently being used in test cases which deal exclusively with tests involving user model (ex. user login, authentication, etc.). In our case user model is not our main test subject, it is just needed to run the code.
2. Using BetaTestUserFactory would be easier to read and follow for a developer without any context at a later stage and can be reused more flexibly in future test cases.
3. The exact same User/UserProfile creation logic seems to be pretty useful as it is in use across multiple test cases throughout the application. The BetaTestUserFactory can be potentialy be used to easily eliminate redundant code in existing test cases.